src/main/java/uk/gov/companieshouse/overseasentitiesapi/controller/OverseasEntitiesController.java

```diff
@@ -161,8 +170,10 @@ public ResponseEntity<Object> createNewSubmissionForSaveAndResume(
             ApiLogger.infoContext(requestId, "createNewSubmissionForSaveAndResume Calling service to create Overseas Entity Submission", logMap);
 
             if (isValidationEnabled) {
+                String passThroughTokenHeader = request.getHeader(ApiSdkManager.getEricPassthroughTokenHeader());
```

Comment:
you could create a private method getPassThroughTokenHeader(request) and put the request.getHeader(ApiSdkManager.getEricPassthroughTokenHeader()); inside it and call it when needed.